server/csp.js

```diff
@@ -4,16 +4,22 @@ const fontAwesomeURL = "https://use.fontawesome.com";
 const defaultSrc = "default-src 'self'";
 const imgSrc = "img-src 'self' data:";
 const fontSrc = `font-src 'self' ${fontAwesomeURL}`;
-const connectSrc = "connect-src 'self' sentry.io";
+const connectSrc = "connect-src 'self' sentry.io https://to9001.b2clogin.com";
```

Comment:
Should this be hard-coded?

One thought. Is it possible to use custom domains with B2C (eg. b2c.virtool.ca). In that case, this URL would also need to be configurable.

Reply:
Ah oops this line should have been deleted, generating a dynamic connect-src is done by the function `generateCSPConnectSrc` regardless of B2C status. Line will be deleted in next push.